src/metrics/misc.ts

```diff
@@ -242,6 +252,55 @@ export class MetadataMetric implements Metric {
       };
     }
 
+
+    // identify game endings
+    let gameEnd = metadata.find(item => item.startsWith('[Termination'));
+    const lastMove = game[game.length - 1].move;
+
+    if (gameEnd === '[Termination "Normal"]') {
+      if (lastMove.originalString.includes('#') || this.chess.isCheckmate()) {
+        this.gameEndings['checkmate'] = (this.gameEndings['checkmate'] || 0) + 1;
+      } else if (result === '[Result "1/2-1/2"]') {
+        this.gameEndings['draw'] = (this.gameEndings['draw'] || 0) + 1;
+      } else if (this.chess.isStalemate()) {
+        this.gameEndings['stalemate'] = (this.gameEndings['stalemate'] || 0) + 1;
+      } else if (this.chess.isInsufficientMaterial()) {
+        this.gameEndings['insufficient material'] = (this.gameEndings['insufficient material'] || 0) + 1;
+      } else {
+        this.gameEndings['resignation'] = (this.gameEndings['resignation'] || 0) + 1;
+      }
+    } else if (gameEnd === '[Termination "Time forfeit"]') {
+      this.gameEndings['time out'] = (this.gameEndings['time out'] || 0) + 1;
+    } 
```

Comment:
thihs seems wrong, stalemate and draw are not mutually exclusive

Reply:
yeah u right stalemate and insufficient material should be nested under draw